application/akka/scala/AkkaQuickstart.scala

```diff
@@ -0,0 +1,85 @@
+//#full-example
+package com.example
+
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import com.example.GreeterMain.SayHello
+
+//#greeter-actor
+object Greeter {
+  final case class Greet(whom: String, replyTo: ActorRef[Greeted])
+  final case class Greeted(whom: String, from: ActorRef[Greet])
+
+  def apply(): Behavior[Greet] = Behaviors.receive { (context, message) =>
+    context.log.info("Hello {}!", message.whom)
+    //#greeter-send-messages
+    message.replyTo ! Greeted(message.whom, context.self)
+    //#greeter-send-messages
+    Behaviors.same
+  }
+}
+//#greeter-actor
+
+//#greeter-bot
+object GreeterBot {
+
+  def apply(max: Int): Behavior[Greeter.Greeted] = {
+    bot(0, max)
+  }
+
+  private def bot(greetingCounter: Int, max: Int): Behavior[Greeter.Greeted] =
+    Behaviors.receive { (context, message) =>
+      val n = greetingCounter + 1
+      context.log.info("Greeting {} for {}", n, message.whom)
+      if (n == max) {
+        Behaviors.stopped
+      } else {
+        message.from ! Greeter.Greet(message.whom, context.self)
+        bot(n, max)
+      }
+    }
+}
+//#greeter-bot
+
+//#greeter-main
+object GreeterMain {
+
+  final case class SayHello(name: String)
+
+  def apply(): Behavior[SayHello] =
+    Behaviors.setup { context =>
+      //#create-actors
+      val greeter = context.spawn(Greeter(), "greeter")
+      //#create-actors
+
+      Behaviors.receiveMessage { message =>
+        //#create-actors
+        val replyTo = context.spawn(GreeterBot(max = 1), message.name)
+        //#create-actors
+        greeter ! Greeter.Greet(message.name, replyTo)
+        Behaviors.same
+      }
+    }
+}
+//#greeter-main
```

Comment:
This whole part can go, right?

Reply:
Indeed, at the moment, both the Java and Scala examples output the text in the console, instead of serving html. I'm trying to address this right now.